content/blog/2022/10/envvar-policy.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Enforcing compliance in your environment variables
+authors:
+- José Guilherme Vanz
+date: 2022-11-01
+---
+
+We're glad to present to Kubewarden users the new [environment-variable-policy](https://github.com/kubewarden/environment-variable-policy). It's able to inspect containers, init containers, and ephemeral containers, and restricts their usage of environment variables by checking their names and values defined under the containers' [`env[*]`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core).
+
+As always, the policy can be found in [Github](https://github.com/kubewarden/environment-variable-policy) and all the  artifacts, including the BOM files, are signed with Sigstore.
+
+## What is so useful about the new environment-variable policy?
+
+This policy complements the recently released  [env-variable-secrets-scanner](https://www.kubewarden.io/blog/2022/10/env-var-secrets/) policy. Both policies focus on validating the environment values provided to Kubernetes Pod objects. In this new policy, users can validate which variables name and values their resources should have. For that, the policy gives four rule operators to control the environment variable defined in the containers. They are:
+
+### anyIn
+
+`anyIn` works as a deny list of your usage of volumes. It checks if any of the `environmentVariables` are in the Pod/Workload resource:
```

Comment:
There's a copy-paste leftover: "volumes".

nitpicking: add an empty line between this paragraph and the code bloc please